tests/secret-providers/hashicorp_vault/hashicorp_vault_test.go

```diff
@@ -426,7 +426,7 @@ spec:
   template:
     spec:
       containers:
-      - image: quay.io/zroubalik/hey
+      - image: ghcr.io/kedacore/tests-hey
```

Comment:
I have updated this file as part of my PR, revert changes in hashicorp file please

Reply:
done